doc/release-notes/6961-payara-upgrade.md

```diff
@@ -60,8 +62,10 @@ In domain1/config/jhove.conf, change the hard-coded /usr/local/glassfish4 path,
 
 (Optional): If you renamed your service account from glassfish to payara or appserver, update the ownership permissions:
 
-	# sudo chown -R appserver /usr/local/payara5/glassfish/domains/domain1
-	# sudo chown -R appserver /usr/local/payara5/glassfish/lib
+	# sudo chown -R payara /usr/local/payara5/glassfish/domains/domain1
```

Comment:
@landreev here you change to a `payara` user, as I had, but per @pdurbin I understand the 5.0 guides will use `appserver`?

Reply:
Ah, no, sorry. The Dataverse 5 guides run Payara under a Unix user called "dataverse". Screnshot below.

<img width="718" alt="Screen Shot 2020-07-31 at 9 46 14 AM" src="https://user-images.githubusercontent.com/21006/89041055-b81cf900-d312-11ea-9fb8-dd64ebb679ad.png">